libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Recognizers/EntityRecognizers/ClientMentionEntityRecognizer.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+using Microsoft.Recognizers.Text;
+using Microsoft.Recognizers.Text.Sequence;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Recognizers
+{
+    /// <summary>
+    /// Recognizer which maps entities passed from the client activity.Entities (@type=mention) into RecognizerResult format.
```

Comment:
if possible add linked docs

```suggestion
    /// Recognizer which maps entities passed from the client activity.Entities (@type=mention) into <see cref="RecognizerResult" /> format.
```